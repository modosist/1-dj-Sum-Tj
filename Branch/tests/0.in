218 14669
132 16903
11 17244
182 17399
51 12587
140 11764
223 16001
155 12802
26 13132
218 14290
81 14167
209 18342
193 13893
128 14174
54 11533
167 14277
126 18493
3 15089
125 16269
196 17037
45 12840
237 17823
78 16740
118 14576
192 14283
239 17859
29 16299
55 11998
79 14145
109 16397
246 12531
177 14126
154 12797
105 17911
170 12978
137 12394
117 16136
220 13396
29 17779
92 16026
97 18547
176 17328
59 14501
214 17011
82 14065
41 15161
167 13382
72 15009
55 11772
163 15607
101 14031
192 18040
20 17297
175 16951
186 11481
85 16353
119 13492
63 18922
142 18967
62 18899
212 16383
159 12426
27 17411
10 11849
84 16894
28 13358
99 14913
39 18525
155 16093
44 13159
249 18783
109 12118
40 18857
153 16379
67 13777
98 18966
21 14282
148 14154
97 12042
112 16140
158 14092
97 11907
85 14131
76 15771
243 16642
73 14381
142 17398
144 14011
100 15813
231 16291
113 15996
146 14258
92 17558
42 15305
203 16381
79 17345
82 14609
178 18761
224 16439
107 13811
108 12853
219 16366
222 13700
33 17010
58 13575
155 13630
90 14500
62 11915
92 14715
90 18709
94 16735
184 15026
156 11591
62 11840
70 11736
167 14041
6 14004
21 17606
172 13972
75 18360
129 18350
228 13553
96 15614
65 15958
118 13946
23 15346
14 14661
179 11873
21 18218
217 14893
126 16655
6 15286
10 14194
22 12485
181 17217
138 13575
247 16578
219 11957
239 16876
20 16214
151 18270
203 11940
43 16461
19 12039
188 13816
168 18386
240 15840
65 12217
241 12048
76 14332
160 17270
107 13659
226 18752
86 15861
230 17313
200 18276
32 16222
34 13678
216 12113
225 13893
188 18625
152 12641
201 12251
127 13352
9 16737
1 11899
191 15948
212 18476
94 14040
163 14690
16 17584
217 13218
73 12584
220 11452
194 13756
154 13890
98 15203
31 13666
121 13740
125 18598
101 15801
206 14944
79 11735
84 11788
71 13378
11 14404
198 13830
214 14772
97 14096
22 15840
137 17563
119 13179
22 15680
55 14256
100 12527
85 18450
105 15175
217 11948
53 15002
184 11891
196 17723
16 14785
157 17866
88 14526
109 14364
171 14238
35 11949
142 14109
199 16914
35 18293
72 15836
153 13722
42 16751
158 15023
196 16470
138 12370
145 18286
24 14020
137 15166
224 11991
156 17500
221 12610
63 16757
214 17528
37 16322
78 11972
118 18956
115 14863
108 17086
49 16795
105 17584
217 12300
97 13947
82 18103
55 19012
188 17654
235 15089
47 14513
218 12662
153 15450
147 16740
122 17689
235 14992
139 17313
213 12957
178 13870
192 11533
100 12062
78 14754
161 12567
113 13238
149 13220
192 14993
172 17922
63 13819
100 13990
179 14070
70 13231
35 14429
163 14746
181 14811
240 18303
190 18336
156 17980
191 17449
117 15812
36 16164
178 12667
36 15735
189 18898
226 14140
209 17609
231 11658
231 12694
221 16056
91 14301
221 15080
1 12991
87 15106
216 11994
249 18305
247 16951
144 17731
89 12243
205 15190
107 16254
205 14514
130 15734
106 14002
45 16809
168 16223
97 18171
151 11500
56 12694
136 11640
97 14735
109 17839
106 11416
245 11548
113 14977
0 0