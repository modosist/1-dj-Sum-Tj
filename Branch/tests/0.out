opt: 1105770
1152584

